docs/administration/protect-sensitive-configuration.md

```diff
@@ -41,3 +41,48 @@ Only users with `Allow modification of sensitive configuration` capability enabl
 
 ![role_allow_modification_sensitive_conf.png](assets%2Frole_allow_modification_sensitive_conf.png)
 
+## Recommended approach to give yourself access to danger zone
+
+1. With a user having the capability “Manage credentials” or “Bypass all capbilities”, go into parameters/security
```

Comment:
1. With a user having the capability “Manage credentials” or “Bypass all capabilities”, go into parameters/security